### PR TITLE
REL-4429 Renaming of IG2 nodding options

### PR DESCRIPTION
**Pull request by cquiroz**
It requires a p1 model bump

### CODE REVIEW

**Review by swalker2m — APPROVED** [review]
Looks great. :+1:

**Comment by cquiroz** [comment]
@swalker2m This was updated after some changes on the task description and also it includes now REL-4420

File: bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/blueprint/SpIgrins2Blueprint.java [hunk 1]
```diff
@@ -25,15 +25,15 @@ public NoddingOption getNoddingOption() {
     }
 
     public SpIgrins2Blueprint(ParamSet paramSet) {
-        noddingOption = Pio.getEnumValue(paramSet, PARAM_NODDING, NoddingOption.KEEP_TARGET_IN_SLIT);
+        noddingOption = Pio.getEnumValue(paramSet, PARAM_NODDING, NoddingOption.NOD_ALONG_SLIT);
```
Comment (swalker2m) [inline bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/blueprint/SpIgrins2Blueprint.java:28]: The blueprints are stored in the science program, so I think we may need to migrate the nodding option unfortunately assuming this was in the 2024A model?

Comment (cquiroz) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/blueprint/SpIgrins2Blueprint.java:28]: That wasn't very clear to me. We haven't imported any IG2 p1 proposal so far

Comment (swalker2m) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/blueprint/SpIgrins2Blueprint.java:28]: Right, I guess this is new for this semester and we don't use `NoddingOption` in the Phase 2 model.  It should be fine. hehe

**Pull request merged**